be/src/exprs/bitmap_function.cpp

```diff
@@ -393,7 +393,7 @@ StringVal BitmapFunctions::bitmap_hash(doris_udf::FunctionContext* ctx,
 
 StringVal BitmapFunctions::bitmap_serialize(FunctionContext* ctx, const StringVal& src) {
     if (src.is_null) {
-        return src;
```

Comment:
Could you explain more about the reason?
Maybe you can write some comment directly in code.
And P0 failed, please check.